tests/test_examples.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import os
+import os.path
+import subprocess
+import re
+
+
+_EXAMPLES_PATH = '../examples/pylint/'
+_EXAMPLE_PREFIX_REGEX = '[CEFRW]\d{4}'
+
+
+def get_test_file_paths():
+	"""Gets all the files from the examples folder for testing."""
+	# Since examples are located elsewhere, change there first. 
+	old_wd = os.getcwd()
+	os.chdir(_EXAMPLES_PATH)
+	cur_wd = os.getcwd()
+	# Now get every file.
+	test_files = []
+	for test_file in os.listdir('.'):
```

Comment:
You can make this nicer and a bit more general if you use `os.walk`. In that case, I don't think you need to change directories either.